operator/v1/types_console.go

```diff
@@ -202,6 +205,30 @@ type AddPage struct {
 	DisabledActions []string `json:"disabledActions,omitempty"`
 }
 
+type PerspectiveState string
+
+// These are valid perspective states. "Enabled" means the perspective is shown.
+// "Disabled" means the Perspective is hidden.
+const (
+	Enabled  PerspectiveState = "True"
+	Disabled PerspectiveState = "False"
```

Comment:
I would expect that the enum value of PerspectiveState.Enabled should be "Enabled".

See https://go.dev/play/p/Xitfcfrdsqj

```suggestion
	Enabled  PerspectiveState = "Enabled"
	Disabled PerspectiveState = "Disabled"
```